src/App.css

```diff
@@ -1,28 +1,45 @@
-.App {
-  text-align: center;
-}
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+* {
+  box-sizing: border-box;
 }
 
-.App-header {
-  background-color: #222;
-  height: 150px;
-  padding: 20px;
-  color: white;
+.board {
+  background: #a0e;
+  display: grid;
+  grid-gap: .5rem;
+  grid-template-columns: repeat(auto-fill, minmax(250px, 300px));
+  grid-template-rows:    repeat(auto-fill, minmax(250px, 300px));
```

Comment:
It seems like the use of the grid display is causing some weirdness with the first row being taller than the other rows (it's getting the 300px height, whereas other rows seems like maybe they're just 250px).